### PR TITLE
open graph tags

### PR DESCRIPTION
**Pull request by thurber**

### CODE REVIEW

File: landing/src/app.html [hunk 1]
```diff
@@ -2,9 +2,23 @@
...
+		<link rel="image_src" href="https://immm-sfa.github.io/msd_uncertainty_ebook/images/ebook_thumbnail_1.png" type="image/png" />
+		<meta property="og:image" content="https://immm-sfa.github.io/msd_uncertainty_ebook/images/ebook_thumbnail_1.png" />
+		<meta property="og:image:secure_url" content="https://immm-sfa.github.io/msd_uncertainty_ebook/images/ebook_thumbnail_1.png" />
+		<meta property="og:image:type" content="image/png" />
+		<meta property="og:description" content="A living guide to sensitivity analysis and diagnostic model evaluation techniques for confronting the computational and conceptual challenges of multi-model, transdisciplinary workflows." />
+		<meta property="og:site_name" content="Addressing Uncertainty in MultiSector Dynamics Research" />
+		<meta property="og:locale" content="en_US" />
+		<meta name="twitter:title" content="Addressing Uncertainty in MultiSector Dynamics Research">
+		<meta name="twitter:description" content="A living guide to sensitivity analysis and diagnostic model evaluation techniques for confronting the computational and conceptual challenges of multi-model, transdisciplinary workflows.">
```
Comment (crvernon) [inline landing/src/app.html:20]: @thurber are these closed?

Comment (crvernon) [reply landing/src/app.html:20]: see 19 and 20

**Review by crvernon — COMMENTED** [review]
see inline @thurber

File: landing/src/app.html (hunk 1 above)
Comment (thurber) [reply landing/src/app.html:20]: it is definitely silly of me to be inconsistent, but luckily meta tags can be left open or closed 😅

**Pull request merged**